libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/factories/action/QuickReplyActionFactory.kt

```diff
@@ -63,16 +64,16 @@ class QuickReplyActionFactory @Inject constructor(
     private fun buildQuickReplyIntent(
         sessionId: SessionId,
         roomId: RoomId,
+        eventId: EventId?,
         threadId: ThreadId?,
     ): PendingIntent {
         val intent = Intent(context, NotificationBroadcastReceiver::class.java)
         intent.action = actionIds.smartReply
         intent.data = createIgnoredUri("quickReply/$sessionId/$roomId" + threadId?.let { "/$it" }.orEmpty())
         intent.putExtra(NotificationBroadcastReceiver.KEY_SESSION_ID, sessionId.value)
         intent.putExtra(NotificationBroadcastReceiver.KEY_ROOM_ID, roomId.value)
-        threadId?.let {
-            intent.putExtra(NotificationBroadcastReceiver.KEY_THREAD_ID, it.value)
-        }
+        eventId?.let { intent.putExtra(NotificationBroadcastReceiver.KEY_EVENT_ID, it.value) }
```

Comment:
With this change, I understand that a quick reply will now appear as a Matrix reply and not a regular message. I am not sure we want to do that. Maybe worth asking for a product confirmation?

Reply:
No, at the moment this does nothing. It passes the event id around, but it's not really used yet, since its only usage would be for replying to a thread (apparently, the new SDK APIs take the latest event id in the thread and retrieve the thread id from it).

Reply:
Are you sure? I see that it is used on NotificationBroadcastReceiverHandler, in particular here: https://github.com/element-hq/element-x-android/blob/166f299c431f8e77ec5c403e6026e1e10b1d65af/libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/NotificationBroadcastReceiverHandler.kt#L165

Reply:
Ah maybe you're right, but it will change if the SDK change so a bit hidden.

Note that I cannot test on my emu, Firebase is broken here. Will test on a real device

Reply:
If the SDK changes (and it will, since I created a PR for it 😅 ) it should only reply to that message in a thread, but not as an explicit reply, so it'll be in the thread but it'll be just another message in it. This is quite confusing, I know...

Reply:
OK, I confirm that the reply sent from the notification is still a regular message.